src/code_ef070_len_3400.c

```diff
@@ -1,6 +1,44 @@
 #include "common.h"
 
 INCLUDE_API_ASM(code_ef070_len_3400, SetCamEnabled);
+/*
```

Comment:
can you leave a single line comment above this specifying what's wrong with the commented func?

Reply:
I just went ahead and removed this one as it's in WIP by me.